requirements.txt

```diff
@@ -1,23 +1,2 @@
-recommonmark==0.6.0
-sphinx==4.4.0
-sphinx-markdown-tables==0.0.12
-sphinx-rtd-theme==0.5.1
-sphinx-autobuild==2020.9.1
-sphinx_click==3.1.0
-click==7.1.2
-texttable==1.6.3
-myst-parser==0.16.1
-# Sphinx 3.3.1 does not require a specific version of docutils
-# but docutils 0.17 changed the output html markup, breaking the RTD theme
-# original issue: https://github.com/sphinx-doc/sphinx/issues/9051
-docutils==0.16
-sphinx-gallery==0.7.0
-sphinx-fontawesome==0.0.6
-matplotlib==3.5.2
-scikit-learn==1.1.1
-pandas==1.4.3
-# Dependencies for the Substrafl FedAvg example on MNIST dataset
-torch==1.11.0
-torchvision==0.12.0
-numpy==1.23.1
-sphinx-copybutton==0.5.0
+-r docs/requirements.txt
+-r examples_requirements.txt
```

Comment:
1. Is everyone ok with splitting the requirements into doc and example ? 
It allows me to delete the docs requirements after building it in the binder docker image
2. what do you think of putting both examples in the examples/ folder and moving the requirements there ? Or having separate requirements.txt for each example ?